include/drivers/timer/nrf_rtc_timer.h

```diff
@@ -11,8 +11,25 @@
 extern "C" {
 #endif
 
+/** @brief Maximum allowed time span that is considered to be in the future.
+ */
+#define NRF_RTC_TIMER_MAX_SCHEDULE_SPAN (BIT(24) - 1)
```

Comment:
this is no longer true as mechanism for scheduling alarms longer than half span got removed.

Reply:
Fixed.